bootstrap.sh

```diff
@@ -445,7 +445,8 @@ build_cmake_command(){
       set_value=OFF
       option_value="${!option}"
       if { [[ "$option_value" = "${FALSE}" ]] && { [[ "$FOUND_VALUE" == "DISABLE"* ]] || [[ "$FOUND_VALUE" == *"OFF" ]]; }; } || \
-         { [[ "$option_value" = "${TRUE}" ]] && [[ "$FOUND_VALUE" == "ENABLE"* ]]; }; then
+         { [[ "$option_value" = "${TRUE}" ]] && [[ "$FOUND_VALUE" == "ENABLE"* ]]; } ||
+         { [[ "$option_value" = "${TRUE}" ]] && [[ "$FOUND_VALUE" != "ENABLE"* ]] && [[ "$FOUND_VALUE" != "DISABLE"* ]]; }; then
```

Comment:
I am a bit bothered by the fact that both `FOOBAR_OFF=FALSE` and `FOOBAR_OFF=TRUE` will set the foobar option to `ON`, although I don't think we have any CMake variables ending with `OFF` at the moment.

Maybe remove the `|| [[ "$FOUND_VALUE" == *"OFF" ]]` logic to avoid the impression that we support variables like this?

Reply:
I think you are right, we went great lengths last time to remove any "negative" options so we should not introduce options with _OFF anyways. I removed the logic to handle `*"OFF"` options.